osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -161,6 +161,8 @@ protected override void OnResuming(Screen last)
         {
             base.OnResuming(last);
 
+            Beatmap.Value.Track.ResetSpeedAdjustments();

```

Comment:
Rather than this, `PlaySongSelect` should set Beatmap.Value.Mods to an empty list.